pkg/controller/ambassadorinstallation/reconcile.go

```diff
@@ -215,6 +225,16 @@ func (r *ReconcileAmbassadorInstallation) Reconcile(request reconcile.Request) (
 		helmValues["image.tag"] = tag
 	}
 
+	if ambObj.Spec.InstallOSS {
+		helmValues["enableAES"] = "false"
+		// We do not want to update image.repository and image.tag if they have already been populated by user supplied
+		// configuration.
+		if len(helmValues["image.repository"]) == 0 && len(helmValues["image.tag"]) == 0 {
+			helmValues["image.repository"] = "quay.io/datawire/ambassador"
+			helmValues["image.tag"] = "1.4.2"
```

Comment:
@inercia do you think I should set `image.tag` here? `image.repository` is the only thing that differs between AES and OSS. If the default `image.tag` is used from the Helm repository, then we don't have to update this field here every time there is a new release. WDYT?

Reply:
Yes, you are right. I think we should only set the `repository`. That way, a Helm chart that matches the `version` in the `AmbassadorInstallation` will be downloaded, and the `image.tag` will be obtained from that chart.